src/test/kotlin/com/pubnub/contract/uuidmetadata/step/GivenSteps.kt

```diff
@@ -7,16 +7,18 @@ import com.pubnub.contract.loadPersonaUUIDMetadata
 import com.pubnub.contract.state.World
 import com.pubnub.contract.uuidmetadata.state.UUIDMetadataState
 import io.cucumber.java.en.Given
+import org.hamcrest.MatcherAssert
+import org.hamcrest.Matchers
 
 class GivenSteps(
     private val world: World,
     private val uuidMetadataState: UUIDMetadataState
 ) {
     @Given("I have a keyset with Objects V2 enabled")
```

Comment:
@Given("I have a keyset with Objects V2 enabled") is common for UUID and Channels. Maybe we could introduce separate GivenSteps file for such cases.  
e.g. in contract `package` create objectv2 package and put into it:
-existing channelmetadata
-existing uuidmetadata
-new common package with GivenSteps that contains:
@Given("I have a keyset with Objects V2 enabled") 
?

Reply:
There's a common place. It's a package `com.pubnub.contract.step`. I guess we can move it there

Reply:
I like this idea.